### PR TITLE
Fix login host list nav bar color issue

### PR DESCRIPTION
**Pull request by harry528tt**
@trooper2013 @bhariharan @ssraza
We've seen an branding issue happened on the "Choose Connection" page. 
After looking into it, I found the issue can be solved by moving the setting up branding part from viewDidLoad() into viewWillAppear().

### CODE REVIEW

**Comment by bhariharan** [comment]
@harry528tt Are you able to reproduce this issue on our sample apps?

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostListViewController.m [hunk 1]
```diff
@@ -175,13 +161,29 @@ - (void)viewWillAppear:(BOOL)animated {
...
-    
+    [self setupBrandingForNavBar];
     [self.tableView reloadData];
     [self resizeContentForPopover];
     // style navigiation bar
     [super viewWillAppear:animated];
 }
 
+- (void)setupBrandingForNavBar {
+    [self.navigationController.navigationBar setTranslucent:false];
```
Comment (bhariharan) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostListViewController.m]: Why are we setting translucent to false?

Comment (trooper2013) [reply libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostListViewController.m]: @harry528tt I agree with @bhariharan.  Why cant this be done in your own subclass?

Comment (harry528tt) [reply libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostListViewController.m]: @bhariharan @trooper2013 I can move the setTranslucent to my own class. But for the remaining part, yes it's also happening on your sample app.

**Comment by harry528tt** [comment]
@bhariharan @trooper2013 I took a look the "[self.navigationController.navigationBar setTranslucent:false];" again, I still think this should be added here instead of my own class. There are two main reasons, 1. It's also happening on the sample app,(Click gear button, the Choose Connection nav bar color got slightly lighter), but because of the current branding color for sample app's nav bar is Salesforce blue, it's very hard to notice. 2. We don't call this class in our code directly, we use SFLoginViewController, so we can't override the viewWillAppear for this class. Please let me know what you think.

**Pull request merged**